reftable/tree.c

```diff
@@ -39,25 +39,20 @@ struct tree_node *tree_search(void *key, struct tree_node **rootp,
 void infix_walk(struct tree_node *t, void (*action)(void *arg, void *key),
 		void *arg)
 {
-	if (t->left) {
+	if (t->left)
 		infix_walk(t->left, action, arg);
-	}
 	action(arg, t->key);
-	if (t->right) {
+	if (t->right)
```

Comment:
I personally don't mind this patch much, but as mentioned, you may hit some resistance. I'd say to give it a try anyway and drop it if people aren't happy with it.

In any case, you should add a short explanation to the commit message what you're doing.

Reply:
> Yeah, in the explanation you might want to say that a following patch is going to port "reftable/tree.c" to the new unit test framework but as a preparatory step you'd like to fix style issues in it.

Actually I thought it was the first commit in the branch, but its the last one. I think it should be the first one instead of the last one, as these kinds of style changes are better as preparatory steps.